econml/_tree_exporter.py

```diff
@@ -25,6 +30,46 @@
     from sklearn.tree.export import _BaseTreeExporter, _MPLTreeExporter, _DOTTreeExporter
 
 
+def _color_brew(n):
```

Comment:
Might as well import this directly above where we're importing the other sklearn internals, but if not please make sure to credit sklearn as the source since this is just copied from there.

Reply:
Yeah I changed the color brew colors. Will add a link in the top to sklearn.